snudown.c

```diff
@@ -10,6 +10,7 @@
 enum snudown_renderer_mode {
```

Comment:
absolutely agree. But, at least it doesn't register modules. I asked Peter about it. His response: "snudown-validator is the snudown.c with a main() function added, so it can be run as a standalone program for fuzz-testing."
I thought about rewriting this part to use validator.c but because I'm not sure how to test it, I decided to leave it as is.